### PR TITLE
add multi-line support in flexibletable

### PR DESCRIPTION
**Pull request by songgao**
r? @patrickxb @mlsteele @maxtaco Thanks!

### CODE REVIEW

File: go/flexibletable/table.go [hunk 1]
```diff
@@ -42,6 +43,71 @@ func (t *Table) Insert(row Row) error {
...
+				switch content := row[i].Content.(type) {
+				case emptyCell:
+					newRow = append(newRow, Cell{
+						Alignment: row[i].Alignment,
+						Frame:     [2]string{"", ""},
+						Content:   row[i].Content,
+					})
+				case MultiCell:
+					notEmpty = true
+					for i := range content.Items {
```
Comment (mlsteele) [inline go/flexibletable/table.go]: Use other name than `i`, like `j`, to avoid shadowing `i`.

File: go/flexibletable/table.go [hunk 2]
```diff
@@ -42,6 +43,71 @@ func (t *Table) Insert(row Row) error {
...
+					}
+					newRow = append(newRow, Cell{
+						Alignment: row[i].Alignment,
+						Frame:     row[i].Frame,
+						Content:   SingleCell{Item: current},
+					})
+				default:
+					// unexported error because this shouldn't happen unless we make a
+					// mistake in code
+					return errors.New("unexpect cell content")
```
Comment (mlsteele) [inline go/flexibletable/table.go]: unexpect -> unexpected

File: go/flexibletable/table.go [hunk 3]
```diff
@@ -42,6 +43,71 @@ func (t *Table) Insert(row Row) error {
...
 }
 
+func (t *Table) breakOnLineBreaks() error {
+
+	// so that there's no need to resize if there's no line break
+	broken := make([]Row, 0, len(t.rows))
+
+	for _, row := range t.rows {
+
+		for notEmpty := true; notEmpty; {
```
Comment (mlsteele) [inline go/flexibletable/table.go]: How about this little change? If it does the same thing, I think it's a little clearer.
```
		notEmpty := true
		for notEmpty {
			newRow := make(Row, 0, len(row))
			notEmpty = false
```

Comment (songgao) [inline go/flexibletable/table.go]: Sounds good; I'll change the `for wrapping := true; wrapping; {` in the same file too then

File: go/flexibletable/table.go (hunk 1 above)
Comment (songgao) [inline go/flexibletable/table.go]: good catch!

**Comment by songgao** [comment]
The comment in `cell.go:24` is external facing, i.e. shown in the doc. `emptyCell` is only used internally during the rendering process. So I'm not sure the user of the package needs to know that. But I've added a comment for the `emptyCell` type. Let me know if it doesn't make sense!

**Comment by mlsteele** [comment]
Oh that makes sense about `cell.go:24`. I'm happy either way you want it then.

**Comment by songgao** [comment]
PTAL @mlsteele Thanks!

**Comment by songgao** [comment]
Thanks Miles!

**Pull request merged**